mappings/icrc/lab-result-mapping.json

```diff
@@ -1,59 +1,31 @@
 {
-  "id": "icrc-labresult-mapping",
+  "id": "icrc-lab-result-mapping",
   "url": "https://datatools4heart.eu/fhir/mappings/icrc/lab-result-mapping",
   "name": "lab-result-mapping",
-  "title": "Mapping of lab results to DT4H Lab Result FHIR profile",
-  "source": [
-    {
-      "alias": "labResults",
-      "url": "https://datatools4heart.eu/icrc/LabResult",
-      "joinOn": ["id_patient"]
-    },
-    {
-      "alias": "encounters",
-      "url": "https://datatools4heart.eu/icrc/Encounter",
-      "joinOn": ["id_patient"]
-    }
-  ],
+  "title": "Mapping of icrc lab results to DT4H Lab Result FHIR profile",
+  "source": [{
+    "alias": "source",
+    "url": "https://datatools4heart.eu/icrc/LabResult"
+  }],
   "context": {
-    "labResultsConceptMap":  {
-      "category": "concept-map",
-      "url": "./lab-results-concept-map.csv"
-    },
     "labResultUnitConversion": {
       "category": "unit-conversion",
       "url": "./lab-results-unit-conversion.csv"
     }
   },
-  "variable": [
-    {
-      "name": "labLoincCode",
-      "language": "text/fhirpath",
-      "expression": "mpp:getConcept(%labResultsConceptMap, lab_code)"
-    },
-    {
-      "name": "relatedEncounter",
-      "language": "text/fhirpath",
-      "expression": "%encounters.where((time_stamp.toString().replace(' ','T') + 'Z').toDate() <= (%context.exec_date.replace(' ','T') + 'Z').toDate() and (time_stamp.toString().replace(' ','T') + 'Z').toDate() + 7 days > (%context.exec_date.replace(' ','T') + 'Z').toDate()).first()"
-    }
-  ],
   "mapping": [
     {
       "precondition": {
-        "name": "loincCodeAndRelatedEncounterExist",
-        "language": "text/fhirpath",
-        "expression": "%labLoincCode.exists() and %relatedEncounter.exists()"
+        "TODO": "add condition if code exists in ValueSet"
```

Comment:
Did not understand this TODO

Reply:
Created a concept-map including DT4H lab result codes, and checked if the code exists in the concept-map.